### ISSUE TITLE
`@stylistic/linebreaks` never rewrites the line break standing in front of a stray semicolon behind a nested block

### ISSUE THREAD

**Issue by firefoxic** [stylelint-stylistic/stylelint-stylistic#372]
Found while fixing #283, by enumerating every raw the three parsers assign and asking which of them the walk still never reaches. This is the fifth, and it is the same failure as the four that issue names: the warning is reported, `--fix` reports a clean pass, the file comes back byte-identical, and a relint reports the same warning again, which is the failure mode of #126.

### What minimal example or steps are needed to reproduce the bug?

A stylesheet whose nested block is followed by a stray semicolon on the line under it, with that one break spelled `\n` while every other break of the file is spelled `\r\n`:

```css
.a {
	.b { c: d }
	;
}
```

PostCSS files the whitespace in front of such a semicolon, along with the semicolon itself, in `raws.ownSemicolon` of the block it follows — `"\n\t;"` here. So do `postcss-scss` and `postcss-less`, each of which reads the shape exactly as plain PostCSS does.

### What minimal configuration is needed to reproduce the bug?

```json
{
  "plugins": [
    "@stylistic/stylelint-plugin"
  ],
  "rules": {
    "@stylistic/linebreaks": "windows"
  }
}
```

### How did you run Stylelint?

```shell
stylelint "**/*.css" --config myconfig.json --fix
```

### What did you expect to happen?

The break rewritten, as a break standing in a selector, in a value, in a set of at-rule parameters, in the text of a comment, or in any of the four raws #283 names already is.

### What actually happened?

It is reported at `2:13`, `--fix` reports no warnings left, and the file is returned exactly as it went in. Linting the output reports `2:13` again. Under `unix` the same stylesheet written the other way round — `a { b { c: d }` and `; }` on the line under it, with `\r\n` in front of the semicolon — is reported at `1:15` and returned unchanged in the same way.

### What is the cause?

The fix at [`lib/rules/linebreaks/index.js`](https://github.com/stylelint-stylistic/stylelint-stylistic/blob/main/lib/rules/linebreaks/index.js) walks the tree and rewrites a fixed list of texts. #283 added to that list what an at-rule writes between its name and its parameters, what a rule, an at-rule or a declaration writes between its two halves, the raw of a `!important` and the whitespace inside the delimiters of a comment. This one it did not name, so it stayed off the list.

The warning, meanwhile, comes from a scan of `root.source.input.css` line by line, which sees every break the file holds, in whichever text of whichever node it stands. So the reading and the writing disagree about which breaks belong to the file — the same disagreement as #270 and #283, over the one text those two left standing.

### Do you have a proposal to fix the bug?

Rewrite it where the walk already rewrites the raws beside it, under the same guard those carry: `raws.ownSemicolon` is absent from every node that is not followed by a stray semicolon. No custom syntax keeps a second copy of it, so a bare write reaches it.

Worth checking twice: the raw holds the semicolon along with the whitespace, so what is rewritten there is whitespace in front of a character of code rather than whitespace on its own — as `raws.between` of a declaration and `raws.afterName` of a Less at-variable already are.
